src/lib/ceramic/client.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+/* eslint-disable require-await */
+
+// Import the Ceramic and Tile document clients
+import { CeramicClient } from '@ceramicnetwork/http-client'
+import { TileDocument } from '@ceramicnetwork/stream-tile'
+
+import { once } from 'lodash'
+
+import Config from '../../config/config'
+import { batch } from '../../lib/utils/async'
+
+const { ceramicNodeURL, ceramicBatchSize } = Config
+
+export const getCeramicClient = once(() => new CeramicClient(ceramicNodeURL))
+
+export const serializeDocument = (document: any) => {
+  const { id, content } = document
+
+  return {
+    ...content,
+    id: String(id),
+  }
+}
+
+export const serializeCollection = (documents: any[]) => {
+  return documents.map(serializeDocument)
+}
+
+export class CeramicModel {
+  static index = null
+
+  static liveIndex = null
+
+  static get ceramic() {
+    return getCeramicClient()
+  }
+
+  static async all(): Promise<TileDocument[]> {
+    const {
+      content: { items = [] },
+    } = await this._getIndex()
+
+    return this._loadEntities(items)
```

Comment:
this seems to load also deleted items, is that correct?

Reply:
No, we're reading index keeping ids then loading each document by id

Deleted documents **aren't indexed**, theirs ids aren't existing in the index. So they won't be loaded

Reply:
do we use another index on top of the append log index?

Reply:
Yes, we've discussed this at the start of the feed development 
There's an index document containing all ids of the feed items sorted by the published date